var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -112,12 +105,17 @@ def configure_args(self):
                             new_header)
 
         # Generic options to compile GCC
-        options = ["--prefix=%s" % prefix, "--libdir=%s/lib64" % prefix,
```

Comment:
Removed `--prefix` as it is already specified in the `AutotoolsPackage`. Other than that, no other changes were made to this list.